### PR TITLE
refactor/1144 - moved calculateSpawnHeading

### LINKED ISSUE #1144: move calculateSpawnHeading to SpawnPattenModel
#worldpeace

### PR DESCRIPTION
**Pull request by felixscheffer**
… from RouteModel to SpawnPattenModel

IMHO spawn related code should be in the SpawnPatternModel and not in the RouteModel.

Resolves #1144

### CODE REVIEW

File: src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js [hunk 1]
```diff
@@ -1000,7 +1000,25 @@ export default class SpawnPatternModel extends BaseModel {
         }
 
         this._positionModel = this._routeModel.waypoints[0].positionModel;
-        this.heading = this._routeModel.calculateSpawnHeading();
+        this.heading = this._calculateSpawnHeading();
+    }
+
+
```
Comment (n8rzz) [inline src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js]: remove extra empty line

**Comment by felixscheffer** [comment]
@erikquinn I really hoped that it would go unnoticed. Honestly, I can't see the need/benefit.

File: src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js (hunk 1 above)
Comment (felixscheffer) [reply src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js]: done

**Comment by felixscheffer** [comment]
I created a ticket and a changelog entry...

**Pull request merged**